src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -20,6 +20,24 @@
 namespace ray {
 namespace gcs {
 
+std::unordered_map<NodeID, std::vector<std::shared_ptr<const BundleSpecification>>>
+GenUnplacedBundlesMap(
```

Comment:
```suggestion
GetUnplacedBundlesPerNode(
```

Reply:
Can you also add a docstring to the helper method?

Reply:
Also prefer to move the method to the unamed namespace. 

```
namespace {
    GetUnplacedBundlesPerNode()
}

namespace ray {
namespace gcs{
}
}
```

Reply:
updated.